test/modules/infiltration/test_infiltration.rb

```diff
@@ -0,0 +1,99 @@
+require_relative '../../helpers/minitest_helper'
+
+class TestInfiltration < Minitest::Test
+  def setup
+    @infiltration = OpenstudioStandards::Infiltration
+
+    # load model and set up weather file
+    template = '90.1-2013'
+    climate_zone = 'ASHRAE 169-2013-4A'
+    std = Standard.build(template)
+    @model = std.safe_load_model("#{File.dirname(__FILE__)}/../../../data/geometry/ASHRAESecondarySchool.osm")
```

Comment:
wish we didnt have to build a template to load an osm. Could use BTAP::FileIO.safe_load_model. Maybe we should move to a standards module?

Reply:
yes, I plan to port this to a new module and change the test. I'll change the test once I make that change in another branch.